tests/unit/src/main/java/org/mule/tck/ZipUtils.java

```diff
@@ -74,33 +73,4 @@ public static void compress(File targetFile, ZipResource[] resources) {
       throw new UncheckedIOException(e);
     }
   }
-
-  /**
-   * Uncompresses the given {@code zipFile} into the {@code targetFile}
-   *
-   * @param zipFile    the source file
-   * @param targetFile the target directory
-   */
-  public static void uncompress(File zipFile, File targetFile) throws IOException {
```

Comment:
Are we sure this method is not being used by anybody else?. Perhaps the gateway team?

Reply:
yes, I added that in the original PR and now I realized I could reused FileUtils.unzip